parcels/grid.py

```diff
@@ -194,10 +194,6 @@ def __set__(self, instance, value):
         instance._cptr.__setitem__(self.name, value)
 
 
-class CGridIndex(Structure):
-    _fields_ = [('xi', c_int), ('yi', c_int), ('zi', c_int)]
-
-
```

Comment:
Correct that this `CGridIndex` structure is now completely gone? We don't need it anymore?

Reply:
There is a CGridIndex in parcels.h, but the formatting of this struct is defined in GridIndex.dtype